CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+-   Title: The tile must conform to [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) format and must sufficiently describe the change. Since PR titles are used to form release notes, titles with generic or non-descriptive content (“Fix build.”, “Add patch.”) are not allowed.
+-   Issue: The first line of the description should contain a reference to the issue that the PR is solving. For example, `fixes #1234` or `resolves #1234`. While this is not required for urgent fixes, it is required for all other PRs so that the issue is clearly tracked and triaged by a core team member.
+-   Description: The first line should be a short, focused summary, while the rest of the description should fill in the details and include any supplemental information a reader needs to understand the change holistically. It might include a brief description of the problem that’s being solved, and why this is the best approach. If there are any shortcomings to the approach, they should be mentioned.
+-   Videos and screenshots: Highlight the changes in the UI. These should be supplemental to the text description, not a replacement for it.
+-   Code Snippets: If the PR is changing an API, include code snippets to highlight the changes. This will expedite a reader's ability to construct the right API calls if they are interested in doing so. These should be supplemental to the text description, not a replacement for it.
+
+### Small Simple Pull Requests
+
+Small, simple pull requests are important because they are:
+
+-   Reviewed more quickly
+-   Reviewed more thoroughly
+-   Less likely to introduce bugs
+-   Easier to merge.
+-   Easier to design well
+-   Less blocking on reviews
+-   Simpler to roll back
+
+Note that reviewers have discretion to reject your change outright for the sole reason of it being too large. Usually they will thank you for your contribution but request that you somehow make it into a series of smaller changes. It can be a lot of work to split up a change after you've already written it, or require lots of time arguing about why the reviewer should accept your large change. It's easier to just write small PRs in the first place.
+
+For a comprehensive guide on how to write small PRs, see this [guide](https://github.com/google/eng-practices/blob/master/review/developer/small-cls.md)
+
+## Contributor License Agreement (CLA)
+
+In order to accept your pull request, we need you to submit a CLA. You only need to do this once. Simply reply to your first pull request with `I have read the CLA Document and I hereby sign the CLA`. Your signature will be recorded automatically in the `cla` branch.
+
+## Code Reviews
+
+A code review is a process where someone other than the author(s) of a piece of code examines that code. Code committed to the codebase is both the responsibility of the author and the reviewer. Code reviews should look at:
+
```

Comment:
It might be helpful to clarify the meaning of an approving review? I think there's many good ways to spell this sort of thing out; here's a rough example of one I liked in the past: "A review approval means both that the content of the PR is relevant and improves `phoenix` and that reviewer feels the core contributing team would be comfortable and capable of maintaining this code if we discover a bug or it needs to change."

Reply:
I'm also hesitant to spell this out as leaving room for some level of subtlety is okay here - where if the quality of reviews goes up because of the core aspects of a review spelled out as below are covered - the confidence in the maintainability / usefulness of said code will be an after thought. I also don't want a a bar per-se as it will ultimately be used as a defensive moat rather than a catalyst. I think we collectively agree with the above sentiment you spell out and am confident we don't need guidance there. I'm going to leave this out for now as I think if we can properly adhere to a simple set of what to look for and some basic guidelines on how to conduct ourselves, things will fall into place.

Reply:
makes sense to me!